test/sql/delete_records.sql

```diff
@@ -14,14 +14,7 @@ BEGIN
     FOR table_elem IN array_lower(aux_tables_array, 1)..array_upper(aux_tables_array, 1) LOOP
       BEGIN
         raise notice 'logging table %', aux_tables_array[table_elem];
-        -- public_repository and role are special because they are region agnostic so we append a region to their names
-        IF aux_tables_array[table_elem] = 'public_repository' THEN
-          -- raise notice '%', format('DELETE FROM public_repository WHERE repository_name LIKE ''%s''', '%' || aws_region);
-          EXECUTE format('DELETE FROM public_repository WHERE repository_name LIKE ''%s''', '%' || aws_region);
-        ELSIF aux_tables_array[table_elem] = 'role' THEN
-          -- raise notice '%', format('DELETE FROM role WHERE role_name LIKE ''%s''', '%' || aws_region);
-          EXECUTE format('DELETE FROM role WHERE role_name LIKE ''%s''', '%' || aws_region);
-        ELSE
+        IF aux_tables_array[table_elem] != 'public_repository' AND aux_tables_array[table_elem] != 'role' THEN
```

Comment:
We should make a task to add some metadata in the `iasql_tables` table on if a table is global or not, so we can automatically filter between these two files.

Reply:
💯

Reply:
We're OSS. I think a Dockerhub image updated with each release makes sense to me?

Reply:
Would it be worth it / possible to have a docker image with dependencies installed that still uses the latest codebase to shave more time off ?

Reply:
Hmm... I don't know if dockerhub would allow that. I'm not even sure how you get an account and/or how many images are allowed for a free account (if they even do that anymore?)

I'm pretty sure each image needs a version tag, but it might just be a string, which would make it *possible* to do what you're suggesting. But again, need to read up on the details here.

However, I don't know how useful that would be, though. For CI we would want to build from scratch no matter what, and for this cleanup job, I don't think it would matter if the image is a few days old?